tests/test_self.py

```diff
@@ -265,8 +265,12 @@ def test_enable_all_works(self) -> None:
         {module}:22:0: C0115: Missing class docstring (missing-class-docstring)
         """
         )
+        # Use an empty pylintrc, otherwise, we'd use the project's pylintrc
+        # which is less stable than the default values.
+        pylintrc = os.path.join(HERE, "data", "pylintrc")
         self._test_output(
-            [module, "--disable=all", "--enable=all", "-rn"], expected_output=expected
+            [module, "--disable=all", "--enable=all", "-rn", f"--rcfile={pylintrc}"],
```

Comment:
This should ideally be done in a separate PR.

Reply:
Turn out the separate PR was required as it took me 4/5 hours to do correctly. https://github.com/PyCQA/pylint/pull/6459